### PR TITLE
[Rule Tuning][SECURITY_SOLUTION] rename Endpoint security

### PR DESCRIPTION
**Pull request by kevinlog**
## Issues
Rename Endpoint Security to drop "Elastic" in accordance with elastic/security-team#186

## Summary
Rename Endpoint Security to drop the "Elastic" term. 

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by kevinlog** [comment]
@rw-access @bm11100 FYI, let me know if you see any issues - we're looking to drop the "Elastic" term from the rules in accordance with elastic/security-team#186

File: etc/ecs_schemas/1.4.0/ecs_flat.json [hunk 1]
```diff
@@ -1390,7 +1390,7 @@
         "name": "intrusion_detection"
       },
       {
-        "description": "Malware detection events and alerts. Use this category to visualize and analyze malware detections from EDR/EPP systems such as Elastic Endpoint Security, Symantec Endpoint Protection, Crowdstrike, and network IDS/IPS systems such as Suricata, or other sources of malware-related events such as Palo Alto Networks threat logs and Wildfire logs.\n",
+        "description": "Malware detection events and alerts. Use this category to visualize and analyze malware detections from EDR/EPP systems such as Endpoint Security, Symantec Endpoint Protection, Crowdstrike, and network IDS/IPS systems such as Suricata, or other sources of malware-related events such as Palo Alto Networks threat logs and Wildfire logs.\n",
```
Comment (brokensound77) [inline etc/ecs_schemas/1.4.0/ecs_flat.json:1393]: Hey @kevinlog, I would revert the changes on the ecs files as these are autogenerated based on the ECS releases and so would have no impact on naming anyway.

File: rules/promotions/elastic_endpoint.toml [hunk 2]
```diff
@@ -7,7 +7,7 @@ updated_date = "2020/07/08"
 [rule]
 author = ["Elastic"]
 description = """
-Generates a detection alert each time an Elastic Endpoint Security alert is received. Enabling this rule allows you to
+Generates a detection alert each time an Endpoint Security alert is received. Enabling this rule allows you to
 immediately begin investigating your Elastic Endpoint alerts.
```
Comment (brokensound77) [inline rules/promotions/elastic_endpoint.toml:11]:
```suggestion
immediately begin investigating your Endpoint alerts.
```

**Review by brokensound77 — COMMENTED** [review]
Thanks @kevinlog, left a couple comments

Also, the rule `name = "Ransomware - Detected - Elastic Endpoint Security"` needs updating too

**Comment by brokensound77** [comment]
@kevinlog can you also change the base branch to `7.9`

**Comment by kevinlog** [comment]
thanks @brokensound77 , I committed the changes

**Comment by bm11100** [comment]
The only other rule we may want to consider changing is `Suspicious Elastic Endpoint Parent Process` -> `Suspicious Endpoint Security Parent Process` and account for it in the description as well. Thoughts? Otherwise, this looks good to me.

**Comment by kevinlog** [comment]
> The only other rule we may want to consider changing is `Suspicious Elastic Endpoint Parent Process` -> `Suspicious Endpoint Security Parent Process` and account for it in the description as well. Thoughts? Otherwise, this looks good to me.

good call @bm11100 - updated

File: rules/windows/defense_evasion_masquerading_as_elastic_endpoint_process.toml [hunk 3]
```diff
@@ -7,14 +7,14 @@ updated_date = "2020/09/30"
 [rule]
 author = ["Elastic"]
 description = """
-A suspicious Elastic endpoint agent parent process was detected. This may indicate a process hollowing or other form of
+A suspicious endpoint agent parent process was detected. This may indicate a process hollowing or other form of
```
Comment (bm11100) [inline rules/windows/defense_evasion_masquerading_as_elastic_endpoint_process.toml:10]:
```suggestion
A suspicious Endpoint Security parent process was detected. This may indicate a process hollowing or other form of
```

File: rules/windows/defense_evasion_masquerading_as_elastic_endpoint_process.toml [hunk 4]
```diff
@@ -7,14 +7,14 @@ updated_date = "2020/09/30"
...
 code injection.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*"]
 language = "kuery"
 license = "Elastic License"
-name = "Suspicious Elastic Endpoint Parent Process"
+name = "Suspicious Endpoint Parent Process"
```
Comment (bm11100) [inline rules/windows/defense_evasion_masquerading_as_elastic_endpoint_process.toml:17]:
```suggestion
name = "Suspicious Endpoint Security Parent Process"
```

**Review by bm11100 — APPROVED** [review]
lgtm

**Review by brokensound77 — APPROVED** [review]
LGTM 👍
